src/js/themes/hpePop.js

```diff
@@ -29,7 +30,118 @@ export const deepMerge = (target, ...sources) => {
   return output;
 };
 
+const getThemeColor = (color, theme) =>
+  typeof theme.global.colors[color] === 'string'
+    ? theme.global.colors[color]
+    : theme.global.colors[color]?.[theme.dark ? 'dark' : 'light'] || color;
+
+const defaultPad = {
+  small: { horizontal: '23px', vertical: '15px', iconOnly: '19px' },
+  medium: { horizontal: '35px', vertical: '19px', iconOnly: '21px' },
+  large: { horizontal: '39px', vertical: '21px', iconOnly: '23px' },
+};
+
+const createButtonSizes = (size) => ({
+  default: {
+    pad: {
+      horizontal: defaultPad[size].horizontal,
+      vertical: defaultPad[size].vertical,
+    },
+  },
+  secondary: {
+    pad: {
+      horizontal: `${parseInt(defaultPad[size].horizontal, 10) - 3}px`,
```

Comment:
Can we add code comment to specify this adjustment is needed to accommodate border?

Reply:
Done.